<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceRecognize</class>
 <widget class="QMainWindow" name="FaceRecognize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FaceRecognize</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49, 71, 65);</string>
    </property>
    <widget class="QWidget" name="widget_LB" native="true">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>380</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(31, 32, 36, 69);</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);

background-color: rgba(31, 32, 36, 69);</string>
      </property>
      <property name="text">
       <string>认证成功</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>480</width>
       <height>480</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="text">
      <string>人脸识别区域</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>widget_LB</zorder>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>320</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	border:1px solid #313236;
	border-radius: 4px;
	background-color: rgb(37, 40, 49);
}

QWidget#titleLB{
	color: rgb(255, 255, 255);
	
	background-color: rgb(61, 61, 61);
}

QWidget#headLB{
	border-radius: 50px;
	
	background-color: rgb(61, 61, 61);
}</string>
    </property>
    <widget class="QLabel" name="titleLB">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>320</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>人脸识考勤别系统</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="headLB">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	border:1px solid #313236;
	border-radius: 4px;
	background-color: rgb(60, 60, 60);
}
QLabel{
	border:none;
	color: #20232A;
	
	font: 16pt &quot;微软雅黑&quot;;
}

QLineEdit{
	border:none;
	font: 16pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	
}</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>工号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>12312321</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	border:1px solid #313236;
	border-radius: 4px;
	background-color: rgb(60, 60, 60);
}
QLabel{
	border:none;
	color: #20232A;
	
	font: 16pt &quot;微软雅黑&quot;;
}

QLineEdit{
	border:none;
	font: 16pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	
}</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>12312321</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	border:1px solid #313236;
	border-radius: 4px;
	background-color: rgb(60, 60, 60);
}
QLabel{
	border:none;
	color: #20232A;
	
	font: 16pt &quot;微软雅黑&quot;;
}

QLineEdit{
	border:none;
	font: 16pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	
}</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>部门</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>12312321</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_7" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	border:1px solid #313236;
	border-radius: 4px;
	background-color: rgb(60, 60, 60);
}
QLabel{
	border:none;
	color: #20232A;
	
	font: 16pt &quot;微软雅黑&quot;;
}

QLineEdit{
	border:none;
	font: 16pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	
}</string>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>时间</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>12312321</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_8" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>320</width>
       <height>480</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	border:1px solid #313236;
	border-radius: 4px;
	background-color: rgb(255, 255, 200);
}

QWidget#titleLB{
	color: rgb(255, 255, 255);
	
	background-color: rgb(61, 61, 61);
}

QWidget#headLB{
	border-radius: 65px;
	
	background-color: rgb(61, 61, 61);
}</string>
     </property>
     <widget class="QLabel" name="titleLB_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>320</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
	
background-color: rgb(61, 61, 61);</string>
      </property>
      <property name="text">
       <string>人脸识考勤别系统</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="headLB_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius: 65px;
	
	background-color: rgb(61, 61, 61);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>301</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget_9" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget{
	border:1px solid #313236;
	border-radius: 4px;
	background-color: rgb(60, 60, 60);
}
QLabel{
	border:none;
	color: #20232A;
	
	font: 16pt &quot;微软雅黑&quot;;
}

QLineEdit{
	border:none;
	font: 16pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	
}</string>
         </property>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>51</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>工号</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_workerid">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>0</y>
            <width>221</width>
            <height>61</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_10" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget{
	border:1px solid #313236;
	border-radius: 4px;
	background-color: rgb(60, 60, 60);
}
QLabel{
	border:none;
	color: #20232A;
	
	font: 16pt &quot;微软雅黑&quot;;
}

QLineEdit{
	border:none;
	font: 16pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	
}</string>
         </property>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>61</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>姓名</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_name">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>0</y>
            <width>221</width>
            <height>61</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>liyong</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_11" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget{
	border:1px solid #313236;
	border-radius: 4px;
	background-color: rgb(60, 60, 60);
}
QLabel{
	border:none;
	color: #20232A;
	
	font: 16pt &quot;微软雅黑&quot;;
}

QLineEdit{
	border:none;
	font: 16pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	
}</string>
         </property>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>51</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>部门</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_work">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>0</y>
            <width>221</width>
            <height>61</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>wh2404</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_12" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget{
	border:1px solid #313236;
	border-radius: 4px;
	background-color: rgb(60, 60, 60);
}
QLabel{
	border:none;
	color: #20232A;
	
	font: 16pt &quot;微软雅黑&quot;;
}

QLineEdit{
	border:none;
	font: 16pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	
}</string>
         </property>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>51</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>时间</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_time">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>0</y>
            <width>221</width>
            <height>61</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>2024</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
